charts/pulsar-operator/README.md

```diff
@@ -46,3 +46,12 @@ To uninstall the `pulsar-operator` chart, execute this command.
 ```
 helm uninstall pulsar-operator -n <k8s-namespace>
 ```
+
+
+## Versioning Convention
+
+`version`: The version of the chart itself, it will change only when there are some changes to the chart itself or `appVersion` bumps a new version.
```

Comment:
```suggestion
`version`: The version of the chart. It will be changed only when there are some changes to the chart or the `appVersion` bumps a new version.
```